package.json

```diff
@@ -29,7 +29,7 @@
     "electron-build-dmg": "node src-script/build-dmg.js",
     "clean:report": "rm -rf .nyc_output || true",
     "tsc": "tsc --build ./tsconfig.json",
-    "test": " npm run test:unit && npm run test:e2e-ci && npm run report",
+    "test": " npm run test:unit && npm run test:e2e-ci",
```

Comment:
I include the explanation in the doc:
If fails due to "cp" or "mv" or "mkdir -p" when executing the above command (npm run report), which is part of 'npm run test', it is because they are supposed to be Linux commands and does not compile in Windows.
Currently we delete the command to exclude it from 'npm run test' to remove failure → **might need to be fixed**

There are too many incompatible commands used when running reports and correcting all of them requires huge efforts and might lead to new bugs or inconsistency, so it is deleted for now.

Reply:
We shouldn't remove commands just because they doesn't work on windows.
The test results (jest and cypress unit tests) are combined and released as part of the Github build job.
It also powers the code change stats within each pull request.

It should be retained via some logical operators so the report runs but do not fail the build if fails
e.g. 

    "test": " npm run test:unit && npm run test:e2e-ci",
    "posttest": "npm run report || true",

Reply:
@YanchengZhou can you add this?

Reply:
I will add these logic instead of directly removing it @jingteng25742

Reply:
In the new commit, I revise the zap-combine-reports.js so that we can do npm run report cross all platforms now and there is no need to delete it anymore